provider/tags.go

```diff
@@ -57,6 +57,40 @@ func applyTags(
 	}
 	if allTags.IsNull() {
 		delete(ret, "tags")
+		delete(ret, "tagsAll")
```

Comment:
This fixes #4080

Reply:
So null tagsAll is replaced with simply missing. That's reasonable I think.